src/styles/index.scss

```diff
@@ -1,3 +1,143 @@
+$stars_size: 16px;
+
 body {
   margin: 0;
+  font-size: 10px;
+  font-weight: 400;
+  font-family: Roboto, sans-serif;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: #{$stars_size};
+  height: #{$stars_size};
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card {
+  display: flex;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.card_info {
+  text-align: left;
+  padding: 0 16px;
+}
+
+.card_imac_img {
+  margin-top: 32px;
+  margin-bottom: 36px;
+  width: 160px;
+  height: 134px;
+  padding: 0 19px;
+}
+
+.card_text_name {
+  color: #060b35;
+  margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card_text_code {
+  color: #616070;
+  margin-top: 4px;
+  margin-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card_stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
```

Comment:
The use of `justify-content: space-between;` here might not be necessary or might not work as expected since there's only one child element `.stars_number` inside `.card_stars`. If there are supposed to be multiple elements spaced out, ensure they are included in the markup. Otherwise, this property might not have any effect.